package.json

```diff
@@ -20,22 +20,18 @@
     "node": ">=18.12.0"
   },
   "dependencies": {
-    "@angular/animations": "^17.3.1",
-    "@angular/cdk": "^17.3.1",
-    "@angular/common": "^17.3.1",
-    "@angular/compiler": "^17.3.1",
-    "@angular/core": "^17.3.1",
-    "@angular/forms": "^17.3.1",
-    "@angular/material": "^17.3.1",
-    "@angular/platform-browser": "^17.3.1",
-    "@angular/platform-browser-dynamic": "^17.3.1",
-    "@angular/platform-server": "^17.3.1",
-    "@angular/router": "^17.3.1",
-    "@angular/ssr": "^17.3.2",
-    "@ngrx/effects": "^17.1.1",
-    "@ngrx/router-store": "^17.1.1",
-    "@ngrx/store": "^17.1.1",
-    "@ngrx/store-devtools": "^17.1.1",
```

Comment:
I removed `@ngrx/*` deps as they are not used and would need an update to v18 otherwise.